test/integration/aws/default/build/flow_log.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_flow_log" "fl-vpc" {
+  log_group_name = "${aws_cloudwatch_log_group.test_log_group.name}"
+  iam_role_arn   = "${aws_iam_role.test_role.arn}"
+  vpc_id         = "${data.aws_vpc.default.id}"
+  traffic_type   = "ALL"
+}
+
+resource "aws_flow_log" "fl-subnet" {
+  log_group_name = "${aws_cloudwatch_log_group.test_log_group.name}"
+  iam_role_arn   = "${aws_iam_role.test_role.arn}"
+  subnet_id      = "${aws_subnet.subnet_01.id}"
```

Comment:
It looks like aws_subnet.subnet_01 is defined in another file.  #2915 will require us to avoid doing that, because we will need to keep the services isolated.  You could look up the default subnet on the default VPC, or make your own subnet here.